.github/workflows/db-backup.yml

```diff
@@ -0,0 +1,55 @@
+name: Public backup to the cloud
+
+on:
+  schedule:
+    - cron: "0 10 * * 1" # At 10:00 on Monday.
+  workflow_dispatch:
+    inputs:
+      home:
+        description: "This can be triggered from the GH page"
```

Comment:
This is because to have a `workflow_dispatch`, you need at least 1 input? Otherwise, could it be deleted?

Reply:
The inputs in this case are not mandatory; they were only for documentation purposes. I just deleted them. 👍